doc/requirements.txt

```diff
@@ -24,3 +24,4 @@ sphinxcontrib-htmlhelp==2.0.0
 sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
+ipython_genutils==0.2.0
```

Comment:
what is this for?

Reply:
> 



> @Emmanuel289 can you look at these comments? ☝️

Sure, I'll take a look

Reply:
> what is this for?

The readthedocs build was failing because of that missing package. So I added that in the list of required packages.